packages/sdk-middleware-auth/src/existing-token.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+import type {
+  Middleware,
+  MiddlewareRequest,
+  MiddlewareResponse,
+  ExistingTokenMiddlewareOptions,
+  Next,
+} from '../../../types/sdk'
+
+export default function createAuthMiddlewareForExistingToken(
+  options: ExistingTokenMiddlewareOptions | string
+): Middleware {
+  return (next: Next) => (
+    request: MiddlewareRequest,
+    response: MiddlewareResponse
+  ) => {
+    const config = options
```

Comment:
Should we bail out immediately when there are no options given?

Reply:
I thought about that. But it will make this middleware unusable in current CLI modules where the accessToken is optional

Reply:
Why would I want to use this middleware if I don't want to use my existing token?